### ISSUE TITLE
Windows 10 doesn't shutdown

### ISSUE THREAD

**Issue by WeirdScienceX** [akorb/SteamShutdown#19]
I just recently came across this program and it is ideal for what I need.

The problem is on Win 10 Version 1803 (Build 17134.112) the shutdown dialogue box appears however the shutdown never commences.  I've never tried it on another version of Windows so I don't know what the expected behaviour is.

![2018-06-25_00-23-59](https://user-images.githubusercontent.com/1296569/41824783-52a78e48-780e-11e8-989a-a2845690bed7.jpg)

**Comment by akorb** [comment]
I tried to replicate your problem. My computer turns off properly, also on Win 10 Version 1803 (Build 17134.112). Did you check the "Complete all downloads" in the SteamShutdown context menu?

**Comment by WeirdScienceX** [comment]
Hi thanks for the reply, I'm sure the fault is down to myself.  I do seem to remember using Winaero at some stage to manipulate shutdown settings but it has been so long since I did it I can't quite remember.

I tried the program on another desktop and a laptop and it performed flawlessly, thanks for such a useful program.

**Comment by akorb** [comment]
I just scrolled through the features of Winaero and couldn't find an option which would break the shut down. Can you try executing the following from the command line? (This will shut down your pc if successful): `shutdown /s /t 0` 

Did it work?

**Comment by WeirdScienceX** [comment]
Yeah the shutdown command works fine.

**Comment by akorb** [comment]
Weird, that's exactly what is executed by my program.
The only thing I can imagine is that there is another `shutdown.exe` before the real one in the search paths.

**Comment by WeirdScienceX** [comment]
I checked the system variables and couldn't see anything but I imagine you're right, something is intercepting calls to shutdown

The only thing I could find in Winaero was the create shutdown shortcut on the desktop which I couldn't imagine causing any issues.

Thanks for all your help anyway, if I get to the bottom of it I will let you know

**Comment by akorb** [comment]
Open command line: `cd "folder with SteamShutdown.exe"`
Afterwards execute `where shutdown`.
Then you can see which shutdown.exe my program will execute.

The shutdown shortcut can be a problem if its name is "shutdown" and is in the same folder as the SteamShutdown.exe or its shortcuts.

**Comment by WeirdScienceX** [comment]
This is what I got:

`C:\Users\d3xp2\Desktop>where shutdown

C:\Windows\System32\shutdown.exe`

**Comment by akorb** [comment]
Then I'm stumped, sorry :/

**Comment by WeirdScienceX** [comment]
Thanks for your help anyway

**Comment by akorb** [comment]
Let me know your solution when you figure it out. :)

**Comment by pointydev** [comment]
Heya,

Not sure if this was ever resolved but I'm kinda having the same issue. Win 10 1903 Insider, I downloaded TERA last night so I could play with a friend, didn't work. You mentioned you were using `shutdown /s /t 0`, so I've got a question: are you executing it as an actual command or are you attempting to emulate keyboard inputs to open WIN+R?

I'd also like to throw in my two cents and suggest running with the `/f` flag, where we're assuming a user has saved their work before starting a download.

Thanks,
Elliott
